azure-functions-java-core/src/main/java/com/microsoft/azure/serverless/functions/annotation/HttpTrigger.java

```diff
@@ -11,18 +11,142 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * <p>The HttpTrigger annotation is applied to Azure functions that will be triggered by a call to the HTTP endpoint that
+ * the function is located at. The HttpTrigger annotation should be applied to a method parameter of one of the following
+ * types:</p>
+ *
+ * <ul>
+ *     <li>{@link com.microsoft.azure.serverless.functions.HttpRequestMessage HttpRequestMessage&lt;T&gt;}</li>
+ *     <li>int</li>
```

Comment:
Replace "int" and "String" with "any native Java types such as int, String, byte[]"